websites/C/Craiyon/dist/metadata.json

```
@@ -0,0 +1,18 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "theusaf",
+		"id": "193714715631812608"
+	},
+	"service": "Craiyon",
+	"description": {
+		"en": "AI model drawing images from any prompt!"
```

```suggestion
		"en": "An AI for model drawing images from any prompt!"
```

---

Neither of these are the official description which can be found in the metadata. 
```suggestion
		"en": "Craiyon is an AI model that can draw images from any text prompt!"
```